examples/data/hypercubes/hypercube-pivot.js

```diff
@@ -0,0 +1,82 @@
+const createSession = require('../../session');
+
+const qlikScript = `
+TempTable:
+Load
+RecNo() as ID,
+RecNo()+1 as ID2,
+Rand() as Value
+AutoGenerate 100
+`;
+
+const properties = {
+  qInfo: {
+    qType: 'my-pivot-hypercube',
+  },
+  qHyperCubeDef: {
+    qDimensions: [
```

Comment:
Could we make this a little bit denser? It's a lot of whitespace in this JSON structure :)